tensorboard/webapp/metrics/views/right_pane/right_pane_test.ts

```diff
@@ -550,6 +550,53 @@ describe('metrics right_pane', () => {
             })
           );
         });
+
+        it('displays expected linked time start step when linked time is not enabled', () => {
+          store.overrideSelector(
+            selectors.getMetricsLinkedTimeSelectionSetting,
+            {
+              start: {step: 20},
+              end: null,
+            }
+          );
+          store.overrideSelector(selectors.getMetricsLinkedTimeEnabled, false);
+          const fixture = TestBed.createComponent(SettingsViewContainer);
+          fixture.detectChanges();
+
+          const el = fixture.debugElement.query(
+            By.css('.linked-time mat-checkbox')
+          );
+          expect(el.nativeElement.textContent).toContain('20');
+
+          store.overrideSelector(
+            selectors.getMetricsLinkedTimeSelectionSetting,
+            {
+              start: {step: 40},
+              end: null,
+            }
+          );
+          store.refreshState();
+          fixture.detectChanges();
+          expect(el.nativeElement.textContent).toContain('40');
```

Comment:
I'd be slightly more comfortable if you could check that they are 'Link by step 20' and 'Link by step 40'.